public/app/features/alerting/unified/components/expressions/AlertCondition.tsx

```diff
@@ -0,0 +1,33 @@
+import { css } from '@emotion/css';
+import React, { FC } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Badge, useStyles2 } from '@grafana/ui';
+
+interface AlertConditionProps {
+  enabled?: boolean;
+  onSetCondition: () => void;
+}
+
+export const AlertCondition: FC<AlertConditionProps> = ({ enabled = false, onSetCondition }) => {
```

Comment:
💬 A simple component to toggle between "make this the alert condition" and "this is the alert condition".